# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1245363246 14400
#      Thu Jun 18 18:14:06 2009 -0400
# Node ID a00e219c402def38b499d32dfeeeae9af24501d7
# Parent  df42106beec886806402c20eefcde3bae8f4cde0
__go_draw_axes__.m: Change the default x11 fontspec from '*,%d' to ',%d'.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2009-06-18  Ben Abbott <bpabbott@mac.com>
+
+	* plot/__go_draw_axes__.m: Change the default x11 fontspec from '*,0'
+	to ',0' to avoid delays searching the fontpath for a font named '*'.
+
 2009-06-18  Daniel Gualberto <daniel@alacer.com.br>
 
 	* polynomial/mpoles.m: Fix infinite loop for a multiplicity of
 	poles at zero. Test added.
 
 2009-06-17  Bertrand Roessli <bertrand.roessli@psi.ch>
 
 	* plot/axis.m: Fix bug for 'axis tight' with multiple surface plots,
diff --git a/scripts/plot/__go_draw_axes__.m b/scripts/plot/__go_draw_axes__.m
--- a/scripts/plot/__go_draw_axes__.m
+++ b/scripts/plot/__go_draw_axes__.m
@@ -1332,17 +1332,17 @@ function __go_draw_axes__ (h, plot_strea
 
   else
     print_usage ();
   endif
 
 endfunction
 
 function fontspec = create_fontspec (f, s, gp_term)
-  if (strcmp (f, "*") && ! strcmp (gp_term, "x11"))
+  if (strcmp (f, "*"))
     fontspec = sprintf ("font \",%d\"", s);
   else
     fontspec = sprintf ("font \"%s,%d\"", f, s);
   endif
 endfunction
 
 function [style, typ, with] = do_linestyle_command (obj, idx, mono,
 						    plot_stream, errbars = "")
